src/controllers/timeOff.js

```
@@ -1,55 +1,264 @@
+  try {
+    const employees = await db.employee.findAll({ attributes: ["empId"] });
+    for (let employee of employees) {
+      this.assignTimeOff(timeOff, employee.empId, year);
+    }
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+exports.assignTimeOff = async (timeOff, empId, year) => {
+  year = year || new Date().getFullYear();
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)